lib/actions/ssh.coffee

```diff
@@ -92,7 +92,7 @@ module.exports =
 			Promise.props
 				username: resin.auth.whoami()
 				uuid: device.uuid		# get full uuid
-				containerId: resin.models.device.getApplicationInfo(device.uuid).get('containerId')
+				containerId: resin.models.device.getApplicationInfo(device.uuid).then(JSON.parse).get('containerId')
```

Comment:
Interesting case. So with `fetch` we have to rely on the headers while `request` may have some heuristics.

I think the problem is here
```
		if responses.length is 1
			[{ statusCode }, body] = responses[0]
			return res.status(statusCode).send(body)
		res.status(207).json(multiResponse(responses))
```

The single response is forwarded but the header is not.